### DISCUSSION TITLE
想顯示png檔案

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#92] [Q&A] [answered]
這是我為插入那張圖片寫的

![螢幕擷取畫面 2021-10-30 201415](https://user-images.githubusercontent.com/90883526/139532677-8c6f41da-2ab5-4063-b1f5-82bdfb55db6a.png)

圖片檔案跟程式都是放在同一個資料夾裡面
但是顯示出來只有這個，但我希望真的能夠呈現出來
然後擷取畫面放上來後還發現自己want還打錯

![螢幕擷取畫面 2021-10-30 201150](https://user-images.githubusercontent.com/90883526/139532682-d9125ede-a6cd-4e81-82d2-37640041e5c6.png)

**Comment by jacky0115** [comment, marked as answer]
試著把圖片放在static資料夾裡面，如果你的html路徑沒動過(一樣是放在template資料夾裡面的話)，把src路徑改成`../static/u.png`

**Reply by bonny123865** [reply]
我原本也有試過放在static資料夾內
但現在才發現自己忘記打/
一線之隔 謝謝你！！
